docs/tutorials/python.md

```diff
@@ -0,0 +1,273 @@
+
+??? tip "Pixi tasks"
+    Tasks are actually a pretty cool pixi feature that are powerful and are run in a cross-platform shell. You can do caching, dependencies and more. Read more about tasks in the [tasks](../features/advanced_tasks.md) section.
+
+```shell
+$ pixi r test
+✨ Pixi task (test): pytest .
+================================================================================================= test session starts =================================================================================================
+platform darwin -- Python 3.12.2, pytest-8.1.1, pluggy-1.4.0
+rootdir: /private/tmp/pixi_py
+configfile: pyproject.toml
+collected 1 item
+
+test_me.py .                                                                                                                                                                                                    [100%]
+
+================================================================================================== 1 passed in 0.00s =================================================================================================
+```
+
+Cool it seems to be working!
+
+### Test vs Default environment
+Interesting thing is if we compare the output of
+the two environments.
+
+```shell
+pixi list -e test
+# v.s. default environment
+pixi list
+```
+
+Is that the test environment has:
+
+```shell
+package          version       build               size       kind   source
+...
+pytest           8.1.1                             1.1 mib    pypi   pytest-8.1.1-py3-none-any.whl
+...
+```
+
+But the default environment is missing this package.
+This way you can finetune your environments to only have the packages that are needed for that environment.
+E.g you could also have a `dev` environment that has `pytest` and `ruff` installed, but you could omit these from the `prod` environment.
+There is a [docker](https://github.com/prefix-dev/pixi/tree/main/examples/docker) example that shows how to set-up a minimal `prod` environment, and copy from there.
+
+## Replacing pypi packages with conda packages
+Last thing, pixi provides the ability for `pypi` packages to depend on `conda` packages.
+Let's add `pygments` to the `pyproject.toml` file. Which is a dependency of the `rich` package.
+
+```shell
+pixi add pygments
+```
+This will add the following to the `pyproject.toml` file:
+
+```toml
+[tool.pixi.dependencies]
+pygments = ">=2.17.2,<2.18"
+```
+
+We can now see that the `pygments` package is now installed as conda package.
+
+```shell
+$ pixi list
+Package          Version       Build               Size       Kind   Source
+...
+pygments         2.17.2        pyhd8ed1ab_0        840.3 KiB  conda  pygments-2.17.2-pyhd8ed1ab_0.conda
+```
+
+This way PyPI dependencies and conda dependencies can be mixed and matched, and seamlessly interoperate.
+
+```shell
+$  pixi r python -c "import pixi_py; pixi_py.say_hello()"
+Hello, World! 🧛
+```
+
+And it still works!
+
+## Conclusion
```

Comment:
I suggested in Ruben tutorial to maybe add a call to action for our users with something like this:

Share your tutorial results or thoughts with our community on social networks using #pixi and @prefix.dev!

Reply:
Added call to action :)